tests/test_postgresql.py

```diff
@@ -113,9 +136,13 @@ def test_checks(self):
         self.check.run()
         metrics = self.check.get_metrics()
 
+        self.assertEquals(len([m for m in metrics if 'table:persons' in str(m[3].get('tags', [])) ]), 11, metrics)
+
+        pg_version = os.getenv('PG_VERSION', '9.4.0')
```

Comment:
Can you use `self.check._get_version` instead, it will test this function too.
Also enables someone to test against local version of PG without having to use the rake CI command.